src/constants/input.js

```diff
@@ -38,7 +38,8 @@ export const achievementSpecs = {
         key: "Achievement",
         placeholder: "",
         label: "Achievement",
-        required: true
+        required: true,
+	disabled: false
```

Comment:
Correct the indentation here and a few similar other places too in this same file